splice_spark/src/test/scala/com/splicemachine/spark/splicemachine/DefaultSourceIT.scala

```diff
@@ -117,9 +117,14 @@ class DefaultSourceIT extends FunSuite with TestContext with BeforeAndAfter with
     assert(newDF.count == 20)
   }
 
-  test("insertion with sampling") {  // DB-9395
+  test("insertion with sampling") {
     val userDir: String = System.getProperty("user.dir")
-    val dataDir = userDir+"/src/test/data/lineitem.csv";
+    val hbaseDir = new File(userDir).getParent + "/platform_it/target/hbase"
```

Comment:
Would it be possible to make hbase-site.xml be part of the classpath of the ITs? I'm worried about other tests failing in the same way
We write hbase-site.xml to platform_it/target when standing up standalone, I think it would be a matter of adding that file to the ITs classpath

Reply:
Done in commit [9fb55e9](https://github.com/splicemachine/spliceengine/pull/3619/commits/9fb55e9202962c914d4ab8aca4884771f49ae943)